.circleci/config.yml

```diff
@@ -1,27 +1,60 @@
+defaults: &golang
```

Comment:
this is an alias: https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/